test/goldenfile/goldenfile.go

```diff
@@ -33,7 +33,7 @@ func (gf *GoldenFile) Read() ([]byte, error) {
 }
 
 func (gf *GoldenFile) Update(data []byte) error {
-	err := ioutil.WriteFile(gf.path, data, 0644)
+	err := ioutil.WriteFile(gf.path, data, 0600)
```

Comment:
gosec complaining about file permissions, goldenfile is used for tests. should be fine